### PR TITLE
CLDC-2093 Merge organisations page - view all merge requests

### PR DESCRIPTION
**Pull request by Dinssa**
This is the first step to creating the merge organisations functionality for support users, it is not the fully functionality. 

This branch creates a new tab section on the organisation page where users (support users) can view all merge requests. It shows all merge requests from the current collection year and all in progress merge requests.

Screenshot 2024-08-08 at 16 12 50

### CODE REVIEW

File: app/controllers/organisations_controller.rb [hunk 1]
```diff
@@ -16,6 +16,7 @@ def index
 
     all_organisations = Organisation.order(:name)
     @pagy, @organisations = pagy(filtered_collection(all_organisations.visible, search_term))
+    @merge_requests = MergeRequest.visible.order(:new_organisation_name)
```
Comment (kosiakkatrina) [inline app/controllers/organisations_controller.rb]: order by the absorbing_organisation.name here?

Comment (Dinssa) [reply app/controllers/organisations_controller.rb]: Yes missed that

File: app/models/merge_request.rb [hunk 2]
```diff
@@ -7,11 +7,25 @@ class MergeRequest < ApplicationRecord
...
 
+  scope :not_merged, -> { where.not(status: "request_merged") }
+  scope :visible, lambda {
+    open_collection_period_start_date = FormHandler.instance.start_date_of_earliest_open_collection_period
+    where(
+      "(status != :merged_status) OR (status = :merged_status AND merge_date >= :open_collection_period_start_date)",
+      merged_status: 4,
+      open_collection_period_start_date:,
+    )
+  }
```
Comment (kosiakkatrina) [inline app/models/merge_request.rb]: Is it possible to reuse some of the existing merged/not merged scopes?
Would some variation of this work?
`merged.where("merge_date >= ?", open_collection_period_start_date).or(not_merged)`

Comment (Dinssa) [reply app/models/merge_request.rb]: Ye it is, will amend it

**Review by kosiakkatrina — APPROVED** [review]

**Pull request merged**